tgui/packages/tgui/interfaces/ExosuitConsole.js

```diff
@@ -0,0 +1,98 @@
+import { toFixed } from 'common/math';
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { AnimatedNumber, Box, Button, NoticeBox, Section, LabeledList } from '../components';
+
+export const ExosuitConsole = props => {
+  const { act, data } = useBackend(props);
+  const {
+    mechs = [],
+  } = data;
+
+  if (!mechs.length) {
+    return (
+      <Section>
+        <NoticeBox textAlign="center">
+          No exosuits detected
+        </NoticeBox>
+      </Section>
+    );
+  }
+
+  return (
+    <Section>
+      {mechs.map(mech => (
+        <Section
+          key={mech.tracker_ref}
+          title={mech.name}
+          buttons={(
+            <Fragment>
+              <Button
+                icon="envelope"
+                content="Send Message"
+                disabled={!mech.pilot}
+                onClick={() => act('send_message', {
+                  tracker_ref: mech.tracker_ref,
+                })} />
+              <Button
+                icon="wifi"
+                content={mech.emp_recharging ? "Recharging..." : "EMP Pulse"}
+                color="bad"
+                disabled={mech.emp_recharging}
+                onClick={() => act('shock', {
+                  tracker_ref: mech.tracker_ref,
+                })} />
+            </Fragment>
+          )}>
+          <LabeledList>
+            <LabeledList.Item label="Integrity">
+              <Box color={mech.integrity <= 30
+                ? 'bad'
+                : mech.integrity <= 70
+                  ? 'average'
+                  : 'good'}>
+                {mech.integrity}%
+              </Box>
+            </LabeledList.Item>
+            <LabeledList.Item label="Charge">
+              <Box color={mech.charge <= 30
+                ? 'bad'
+                : mech.charge <= 70
+                  ? 'average'
+                  : 'good'}>
+                {mech.charge ? mech.charge + "%" : "Not Found"}
+              </Box>
+            </LabeledList.Item>
+            <LabeledList.Item label="Airtank">
+              {mech.airtank
+                ? <AnimatedNumber
+                  value={mech.airtank}
+                  format={value => toFixed(value, 2) + ' kPa'} />
+                : "Not Equipped"}
+            </LabeledList.Item>
+            <LabeledList.Item label="Pilot">
+              {mech.pilot ? mech.pilot : "None"}
```

Comment:
Write `mech.pilot || 'None'` to not repeat yourself.

Reply:
Done.